### PR TITLE
Fixes #20345 - add back button on host info page

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

File: app/helpers/hosts_helper.rb [hunk 1]
```diff
@@ -301,48 +301,51 @@ def state(s)
...
-                                    }
-              )
-            end
-        ),
-        if host.supports_power?
-          button_group(
-              link_to(_("Loading power state ..."), '#', :disabled => true, :class => 'btn btn-default', :id => :loading_power_state)
+      button_group(
+        link_to(_("Back"), hosts_path, :class => 'btn btn-default')
+      ),
```
Comment (tbrisker) [inline app/helpers/hosts_helper.rb:306]: These three lines are the only actual change, the rest is correcting alignment.

**Comment by lzap** [comment]
Aren't back buttons UX evil? I am good with the change tho :-) What's Roxanne's nick again?

**Comment by ohadlevy** [comment]
@tbrisker do you mind adding a before / after image that @rohoover can have a look? thanks!

**Comment by tbrisker** [comment]
Before:
screenshot from 2017-07-20 10-14-49

After:
screenshot from 2017-07-20 10-14-49

**Review by dLobatog — COMMENTED** [review]
I tend to agree with @lzap , probably it's best to save the search in a cookie or similar so that on future accesses to "All hosts" (clicking the item from the menu) the search is saved. Personally I found back and forward buttons very redundant (all browsers have that..)

**Comment by lzap** [comment]
I noticed this is backport request, if you need this than we can simply merge this and create refactor ticket, that would definitely work for me.

**Comment by timogoebel** [comment]
Just my two cents: I think breadcrumbs would be better than a "Back" button.

**Comment by tbrisker** [comment]
@dLobatog - I wonder if cookies won't be a bit overkill for this? other "back" buttons don't have this functionality.
@timogoebel - I agree, but currently we already have a "back" button in several other pages and this will be consistent with them. Adding breadcrumbs is definitely the long-term solution, but i'd rather be consistent now than add different implementations on different pages or refactor all the other pages for this.

**Comment by ohadlevy** [comment]
@rohoover can you ack/nack please? thanks!

**Comment by Rohoover** [comment]
I'm fine with adding the back button (as a consistent temporary solution) so long as we intend to nix all back buttons for breadcrumbs instead :)

**Pull request merged**